bitnami/cert-manager/values.yaml

```
@@ -201,6 +201,18 @@ controller:
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   ##
   podAnnotations: {}
+  ## @param controller.dnsPolicy Pod DNS policy
+  ## Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  ##
+  dnsPolicy: ClusterFirst
+  ## @param controller.dnsConfig Pod DNS config
+  ## Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config
+  ## E.g:
+  ## dnsConfig:
+  ##   nameservers:
+  ##     - "1.1.1.1"
+  ##
+  dnsConfig: []
```

```suggestion
  dnsConfig: {}
```

This setting is an object.